web3j/src/main/java/com/vmware/ethereum/service/SecureTokenApi.java

```diff
@@ -84,10 +101,49 @@ private SecurityToken deploy() {
                 initialSupply)
             .send();
 
-    securityToken.getTransactionReceipt().ifPresent(receipt -> log.info("Receipt: {}", receipt));
+    securityToken
+        .getTransactionReceipt()
+        .ifPresent(
+            receipt -> {
+              log.info("Receipt: {}", receipt);
+              contractAddress = receipt.getContractAddress();
+            });
     return securityToken;
   }
 
+  /** Transfer token for deferred polling. */
+  public String transferQueued(
+      CountDownLatch countDownLatch, HashMap<String, Instant> txHashTime, MetricsService metrics) {
```

Comment:
What is the purpose of having countDownLatch and metrics in this method?

Reply:
unused parameters removed